templates/unit_file.epp

```diff
@@ -5,6 +5,7 @@
  Optional[Hash] $timer_entry,
 | -%>
 <% if $unit_entry { -%>
+# Deployed with puppet
```

Comment:
I'm wondering if this comment should be above line 7? but with all the braces the template is hard to read :D

Reply:
Should be it and it clearly needed a test.